### ISSUE TITLE
Base/Config/Validate panic

### ISSUE THREAD

**Issue by daffinito** [newrelic/newrelic-diagnostics-cli#205]
Base/Config/Validate is panicking on a newrelic.js file:

```
Executing following diagnostic task suites: Node Agent

Check Results
-------------------------------------------------
Info     Base/Env/CollectEnvVars [Gathered Environment variables of current shell.]
Success  Base/Config/Collect
panic: runtime error: index out of range [1] with length 1

goroutine 20 [running]:
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.formatJs({0x14000312980, 0x71})
	/Users/daffinito/code/newrelic-diagnostics-cli/tasks/base/config/validate.go:319 +0x568
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.parseJs({0x1029bd1a0?, 0x140004e7600?})
	/Users/daffinito/code/newrelic-diagnostics-cli/tasks/base/config/validate.go:364 +0x58
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.processConfig({{0x1400002635b?, 0x13?}, {0x14000026320?, 0x102bf4940?}})
	/Users/daffinito/code/newrelic-diagnostics-cli/tasks/base/config/validate.go:187 +0x2c8
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.BaseConfigValidate.Execute({}, {0x140003a1998?}, 0x102554578?)
	/Users/daffinito/code/newrelic-diagnostics-cli/tasks/base/config/validate.go:102 +0x22c
main.processTasks({0x1400023ac90?}, {0x0, 0x0, 0x0?}, 0x140001993a0?)
	/Users/daffinito/code/newrelic-diagnostics-cli/processTasks.go:121 +0x818
created by main.main
	/Users/daffinito/code/newrelic-diagnostics-cli/core.go:128 +0x930
```

**Comment by kenguest** [comment]
I got something very similar when running the diagnostics tool on Windows, through Git Bash
Mentioning this here so you can determine if the same fix addresses this problem.
Thanks :-)

We've found a file that may contain secure information: C:\dev\code\foo\AppBase\node_modules\cordova-ios\bin\templates\project\__TEMP__.xcodeproj\project.pbxproj
Include this file in nrdiag-output.zip?
Choose 'y' or 'n', then press enter: n
Success             Base/Config/Collect
panic: runtime error: index out of range [1] with length 1

goroutine 7 [running]:
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.formatJs({0x11aaa000, 0x580c})
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:319 +0x7ad
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.parseJs({0x658c10, 0x11d98a10})
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:364 +0x8f
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.processConfig({{0x11d82567, 0xb}, {0x11d82500, 0x67}})
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:187 +0x373
github.com/newrelic/newrelic-diagnostics-cli/tasks/base/config.BaseConfigValidate.Execute({}, {0x11bce040}, 0x11bce060)
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/tasks/base/config/validate.go:102 +0x1d0
main.processTasks({0x1197c0a0}, {0x0, 0x0, 0x0}, 0x11819a80)
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/processTasks.go:121 +0x948
created by main.main
        /home/runner/work/newrelic-diagnostics-cli/newrelic-diagnostics-cli/core.go:128 +0xb81

ken@laptop MINGW64 /c/dev/code/tramex/TramexAppBase (master)
$

**Comment by daffinito** [comment]
hi @kenguest, thank you for adding details to this! We've released version 3.2.4 of the CLI which should resolve this issue. Please give it a try and let us know if you are still seeing any issues!
